pkgs/by-name/my/myks/package.nix

```
@@ -0,0 +1,47 @@
+{ lib, buildGoModule, fetchFromGitHub, testers, installShellFiles, myks, }:
+
+buildGoModule rec {
+  pname = "myks";
+  version = "3.4.4";
+
+  src = fetchFromGitHub {
+    owner = "mykso";
+    repo = "myks";
+    rev = "v${version}";
+    hash = "sha256-PaA8j4BWijhPR3DTZ0nnO54v0Uj/DpFdJpofseTA1+A=";
+  };
+
+  vendorHash = "sha256-A30SyqgAeYwgiYLZF9M3iW2u8JXPU6ozUThziCmSRgU=";
+
+  subPackages = ".";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${version}"
+    "-X main.commit=nixpkg-${src.rev}"
+    "-X main.date=1970-01-01"
+  ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  CGO_ENABLED = 0;
+
+  passthru.tests.version = testers.testVersion { package = myks; };
+
+  postInstall = ''
```

```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

See https://github.com/NixOS/nixpkgs/issues/308283